segmentation_processing/README.md

```diff
@@ -0,0 +1,45 @@
+# STEGO Staging
+This directory holds files related to working with the STEGO model provided by ASU. STEGO model implementation by Blake Harrison. Server script modifications by Jacob Lichtefeld.
+
+This server runs in a Python 3.8 environment.
+
+## Creating the Segmentation Environment Locally
+
+To create a local environment for this directory either make a conda environment or a python venv then:
+
+```bash
+# activate your environment
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+## Cluster Configuration
+
+With `segmentation_processing` as the working directory, run the following command, which will create a virtual environment (`venv`) with the necessary dependencies in the working directory, as well as download the STEGO model.
+
+```bash
+PATH="/nas/gaia/adam/shared/local_python/bin/:$PATH" make install
+```
+
+### Creating and upgrading a local Python instance
+
+The easiest way we've found is to create a local Python instance is with Conda, which also has the advantage of using a pre-compiled `python` executable that has no linking issues.
```

Comment:
Is this an alternative to the steps above? Does it need to be done beforehand? Please clarify this

Reply:
This is the instructions to setup a local python instance on the cluster which isn't linked against any system library. 

I followed the Lestat documentation trend of putting the less-frequently followed setup steps later in the readme. This is intended to mostly be a documentation of how we configured our server, any individual deployment by someone in the future has other details to work out which I can't document.

Reply:
@lichtefeld could you add flow sentences under each header saying which steps are less-frequently followed and why? I think it would also be helpful to put a statement to this effect:

> any individual deployment by someone in the future has other details to work out which I can't document.

at the top of the README.